src/ui/app/cmp-app.js

```diff
@@ -1,16 +1,61 @@
 import React, { Component } from 'react';
 
+let latitude = 0;
+let longitude = 0;
+
 class App extends Component {
+  // getGeoData() {
+  //   const url =
+  //     'https://api.openweathermap.org/data/2.5/weather?lat=50.431782&lon=30.516382&appid=e355c588a9911a706b20a89a2fd6f8e5';
```

Comment:
The API key is public. Now it can be used by attackers. Never commit code that has API key